### PR TITLE
docs(business-partner-kit):icon format changed to SVG

### PR DESCRIPTION
**Pull request by Unsharm**
## Description
In this pull request, I have changed the format of icon's image in adoption view, operation view and changelog of the Business Partner kit from .PNG to .SVG.

## Pre-review checks

Please ensure to do as many of the following checks as possible, before asking for committer review:

- [X] DEPENDENCIES are up-to-date. [Dash license tool](https://github.com/eclipse/dash-licenses). Committers can open IP issues for restricted libs.
- [X] [Copyright and license header](https://eclipse-tractusx.github.io/docs/release/trg-7/trg-7-02) are present on all affected files

### CODE REVIEW

File: docs-kits/kits/Business Partner Kit/Documentation BPDM/bpdm_arc42.md [hunk 1]
```diff
@@ -5,7 +5,7 @@ description: 'Architecture documentation (arc42)'
 sidebar_position: 1
 ---
 
-![Business partner kit banner](@site/static/img/BPKitIcon.png)
+![Business partner kit banner](/img/kit-icons/bp-kit-icon.svg)
```
Comment (Siegfriedk) [inline docs-kits/kits/Business Partner Kit/Documentation BPDM/bpdm_arc42.md:8]: Is that correct that you reference this directly?

And what happend to the BPKitIcon.png file?

**Review by Siegfriedk — CHANGES_REQUESTED** [review]

File: docs-kits/kits/Business Partner Kit/Documentation BPDM/bpdm_arc42.md (hunk 1 above)
Comment (Unsharm) [reply docs-kits/kits/Business Partner Kit/Documentation BPDM/bpdm_arc42.md:8]: > Is that correct that you reference this directly?
> 
> And what happend to the BPKitIcon.png file?

> @Siegfriedk  Yes, It is correct. 
>The changes in format will be take effect in the next version. I have created a new folder named "kit-icons" where all the icon images are stored in SVG format.

Comment (Siegfriedk) [reply docs-kits/kits/Business Partner Kit/Documentation BPDM/bpdm_arc42.md:8]: Just to be clear right: You are aware that docusourus uses @site to determine a relative path if the page would deloyed differently and you use the static path without any @site fully aware what it means?

Comment (SujitMBRDI) [reply docs-kits/kits/Business Partner Kit/Documentation BPDM/bpdm_arc42.md:8]: Hello @Siegfriedk, if you refer [Docusaurus Docs](https://docusaurus.io/docs/static-assets#in-markdown) on usage of static assets under markdown file. It is clearly mentioned the image references should not use `@site`, but just need to use absolute paths and we checked rendering is not being affected even if we provide static path.

**Review by maximilianong — APPROVED** [review]

**Review by Siegfriedk — APPROVED** [review]

**Pull request merged**